NDVALPHA-UPDATED-HYPERV-HOST-REGISTRATION (#94627)
Azure/azure-content: articles/migrate/tutorial-migrate-hyper-v.md

1. Register the Hyper-V host to Azure Migrate.

> [!NOTE]

> > If your Hyper-V host was previously registered with another Azure Migrate Project that you are not longer using or deleted, you will need to de-register it from that Project and register it in the new one. Follow the [Remove servers and disable protection](https://docs.microsoft.com/en-us/azure/site-recovery/site-recovery-manage-registration-and-protection?WT.mc_id=modinfra-39236-thmaure#unregister-a-connected-configuration-server) guide to so.

Reviewer: ```suggestion
> If your Hyper-V host was previously registered with another Azure Migrate project that you are no longer using or have deleted, you'll need to de-register it from that project and register it in the new one. Follow the [Remove servers and disable protection](https://docs.microsoft.com/en-us/azure/site-recovery/site-recovery-manage-registration-and-protection?WT.mc_id=modinfra-39236-thmaure#unregister-a-connected-configuration-server) guide to do so.
```
